src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+import core.basesyntax.controller.ConsoleHandler;
+import core.basesyntax.exception.DaoAnnotationNotFound;
+import core.basesyntax.lib.Injector;
+import java.lang.reflect.InvocationTargetException;
```

Comment:
classes from `java.lang` package are available without importing
so we can remove this line

Reply:
@matviichuk92 checked this - this class is not available without importing
sorry, my bad